ci/ansible/roles/pulp-populate/tasks/main.yaml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Install packages for population task
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=latest"
+  with_items:
+    - python34
+    - python34-setuptools
+    - gcc
+
+- name: Ensure services are running
+  service: "name={{ item }} state=started"
+  with_items:
+    - httpd
+    - pulp_celerybeat
+    - pulp_resource_manager
+    - pulp_streamer
+    - pulp_workers
+    - qpidd
+
+- name: Create config directory
+  file: path="{{ ansible_user_dir }}/.config/pulp_smash" state=directory
+
+- name: Ensure a Pulp Smash configuration file is present
+  template:
+    src: settings.j2
+    dest: "{{ ansible_user_dir }}/.config/pulp_smash/settings.json"
+
+- name: Populate the server
+  script: populate.sh "{{ test_results_dir }}"
```

Comment:
This script can be dropped if the following playbook is used:

```yaml
---
- name: Install packages for population task
  action: "{{ ansible_pkg_mgr }} name={{ item }} state=latest"
  with_items:
    - python34
    - python34-setuptools
    - gcc
    - git

- name: Ensure services are running
  service: "name={{ item }} state=started"
  with_items:
    - httpd
    - pulp_celerybeat
    - pulp_resource_manager
    - pulp_streamer
    - pulp_workers
    - qpidd

- name: Create a directory for virtualenvs
  file:
    path: "{{ ansible_user_dir }}/.venvs"
    state: directory

- name: Create a virtualenv
  command: python3 -m venv "{{ ansible_user_dir }}/.venvs/pulp-migrate"
  args:
    creates: "{{ ansible_user_dir }}/.venvs/pulp-migrate"

# We fetch a zip with https://, because something asks for a password when this
# is used: git+https://git@github.com/PulpQE/pulp-migrate.git#egg=pulp-migrate
- name: Install pytest and pulp-migrate into the virtualenv
  shell: >
    source "{{ ansible_user_dir }}/.venvs/pulp-migrate/bin/activate" &&
    pip install pytest https://github.com/PulpQE/pulp-migrate/archive/master.zip#egg=pulp-migrate

- name: Create directory for pulp smash configuration file
  file:
    path: "{{ ansible_user_dir }}/.config/pulp_smash"
    state: directory

- name: Create pulp smash configuration file
  template:
    src: settings.j2
    dest: "{{ ansible_user_dir }}/.config/pulp_smash/settings.json"

- name: Populate the server
  shell: >
    source "{{ ansible_user_dir }}/.venvs/pulp-migrate/bin/activate" &&
    pytest --junit-xml "{{ test_results_dir }}/populate.test.report.xml" --pyargs pulp_migrate.populate
```